src/control/cmd/daos/main.go

```diff
@@ -206,6 +206,15 @@ or query/manage an object inside a container.`
 		return nil
 	}
 
+	// Configure DAOS client logging to stderr if no log file
```

Comment:
It wasn't my first choice, but it's necessary to do it this way in order to work with how logging is configured in libdaos: https://github.com/daos-stack/daos/blob/master/src/common/debug.c#L160 ... If `D_LOG_FILE` is not set, then all logging goes to `stdout`, and this will interfere with JSON output (as described in the ticket).

Reply:
Oups, missed the thing that it was limited to the daos command.
This solution is perfectly OK from my understanding of the daos command usage.